integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestLoadOptions.scala

```diff
@@ -35,6 +36,15 @@ class TestLoadOptions extends QueryTest with BeforeAndAfterAll{
     sql("drop table if exists TestLoadTableOptions")
   }
 
+  override def beforeEach(): Unit = {
+    sql("drop table if exists carriage_return_in_string")
```

Comment:
other previous test cases before this PR, don't need this table. so adding before each is not useful for it.
Suggest to keep in inside testcase and combine multiple new test cases.

Reply:
> other previous test cases before this PR, don't need this table. so adding before each is not useful for it.
> Suggest to keep in inside testcase and combine multiple new test cases.

done